### PR TITLE
Add install piped on ECS Fargate guide

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  "taskDefinition": "{PIPED_TASK_DEFINITION_ARN}",
+  "deploymentConfiguration": {
+    "maximumPercent": 100,
+    "minimumHealthyPercent": 100
+  },
+  "schedulingStrategy": "REPLICA",
+  "launchType": "FARGATE",
+  "networkConfiguration": {
+    "awsvpcConfiguration": {
+      "assignPublicIp": "ENABLED", # This is need to enable ECS deployment pull piped container image.
```
Comment (nakabonne) [inline docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md:135]: JSON doesn't support any kind of comment, but so what!
```suggestion
      "assignPublicIp": "ENABLED", # This is need to enable ECS deployment to pull piped container images.
```

**Comment by nakabonne** [comment]
LGTM except for the nits comment

File: docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  #     privateKeyData: {BASE64_ENCODED_PRIVATE_KEY}
+  #     publicKeyData: {BASE64_ENCODED_PUBLIC_KEY}
+```
+
+- Create a new secret in [SecretManager](https://aws.amazon.com/secrets-manager/) to store above configuration data securely
+
+```console
+aws secretsmanager create-secret --name PipedConfig \
+  --description "Configuration of piped running as ECS Fargate task" \
+  --secret-string file://piped-config.yaml
```
Comment (nghialv) [inline docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md:70]: Is this format required by `aws` command?
Or just our convention to tell users that this is a file path.

Comment (nghialv) [reply docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md:70]: I meant the format of the last one:`file://file-name` .

File: docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+```json
+{
+  "family": "piped",
+  "executionRoleArn": "{PIPED_TASK_EXECUTION_ROLE_ARN}",
+  "containerDefinitions": [
+    {
+      "name": "piped",
+      "image": "gcr.io/pipecd/piped:{{< blocks/latest_version >}}",
+      "command": [
+        "echo $configData > /etc/piped/config.yaml; piped --config-file=/etc/piped/config.yaml --metrics=true"
```
Comment (nghialv) [inline docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md:88]: Metrics is already enabled by default so I guess we can omit its arg in this command.

File: docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md (hunk 2 above)
Comment (khanhtc1202) [reply docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md:70]: it's the AWS command's args convention 👀 https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-parameters-file.html

File: docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md (hunk 1 above)
Comment (khanhtc1202) [reply docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md:135]: Yes, it's. This code block comments are for explanation and we can expect users to understand that Json's comments are not allowed, as you said 😄

File: docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md (hunk 2 above)
Comment (nghialv) [reply docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md:70]: Thanks for the link. Didn't know about that. 👍

File: docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md (hunk 3 above)
Comment (khanhtc1202) [reply docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md:88]:
```suggestion
        "echo $configData > /etc/piped/config.yaml; piped --config-file=/etc/piped/config.yaml"
```

**Comment by nghialv** [comment]
Nice. Thank you.
/approve

**Pull request merged**